src/vulkan-cts-runner.cc

```diff
@@ -338,6 +341,10 @@ bool process_block(Context &ctx, unsigned thread_id) {
       test_idx = it->second;
       indices.erase(it);
       test_active = true;
+    } else if (string_matches(line, "FATAL ERROR: ")) {
```

Comment:
The comment I gave during XDC is that we probably should only dump this outside of any test, to avoid any test crashes that include deqp catching the error to dump output.

Reply:
Yeah, sounds good.  I've got a bunch more stuff (including a testsuite) I came up with on the flights home, which I need to finish off.